readthedocs/projects/managers.py

```diff
@@ -4,4 +4,11 @@
 class HTMLFileManager(models.Manager):
 
     def get_queryset(self):
-        return super(HTMLFileManager, self).get_queryset().filter(is_html=True)
+        return super(HTMLFileManager, self).get_queryset().filter(name__endswith='.html')
+
+
+class ImportedFileManager(models.Manager):
+
+    def get_queryset(self):
```

Comment:
I don't think we should exclude them here. This will change the logic in other downstream code which is using `ImportedFile`, which we don't want to do.